{"name": "test_login_page", "status": "failed", "statusDetails": {"message": "AssertionError: Expected error message 'Epic sadface: Username and password do not match any user in this ' but got 'Epic sadface: Username and password do not match any user in this service'\nassert 'Epic sadface... this service' == 'Epic sadface...user in this '\n  \n  Skipping 55 identical leading characters in diff, use -v to show\n  - er in this \n  + er in this service\n  ?            +++++++", "trace": "self = <Tests.test_negative_login.TestLogin object at 0x104f76490>, driver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"a5a8b16e47038fe5ead2f0925d9fb7fc\")>\n\n    @pytest.mark.sanity\n    def test_login_page(self, driver):\n        Uname = TestData.invalid_Uname\n        password = TestData.Invalid_pass\n    \n        INVALID_LOGIN = LoginPage(driver)\n        INVALID_LOGIN.do_login(Uname,password)\n        try:\n            error_message = INVALID_LOGIN.invalid_cred_message_verify()\n        except NoSuchElementException:\n            pytest.fail(\"element has not found\")\n        expected_error_message = \"Epic sadface: Username and password do not match any user in this \"\n    \n>       assert error_message == expected_error_message, f\"Expected error message '{expected_error_message}' but got '{error_message}'\"\nE       AssertionError: Expected error message 'Epic sadface: Username and password do not match any user in this ' but got 'Epic sadface: Username and password do not match any user in this service'\nE       assert 'Epic sadface... this service' == 'Epic sadface...user in this '\nE         \nE         Skipping 55 identical leading characters in diff, use -v to show\nE         - er in this \nE         + er in this service\nE         ?            +++++++\n\ntest_negative_login.py:29: AssertionError"}, "start": 1717020269395, "stop": 1717020269636, "uuid": "0cbf4df5-c43f-4a59-ae0a-8258570313db", "historyId": "33087646b8c05de395d0c12a2947dba3", "testCaseId": "33087646b8c05de395d0c12a2947dba3", "fullName": "Tests.test_negative_login.TestLogin#test_login_page", "labels": [{"name": "tag", "value": "sanity"}, {"name": "tag", "value": "@pytest.mark.usefixtures('driver')"}, {"name": "parentSuite", "value": "Tests"}, {"name": "suite", "value": "test_negative_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "Chethans-MacBook-Pro.local"}, {"name": "thread", "value": "12046-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "Tests.test_negative_login"}]}